reactos/dll/win32/browseui/basebarsite.cpp

```diff
@@ -373,28 +510,67 @@ HRESULT STDMETHODCALLTYPE CBaseBarSite::Exec(const GUID *pguidCmdGroup, DWORD nC
     {
         switch (nCmdID)
         {
+            case 0:
+                //update band info ?
             case 1:     // insert a new band
                 if (V_VT(pvaIn) != VT_UNKNOWN)
```

Comment:
Is `VT_UNKNOWN` the only type accepted?
Maybe `VT_UNKNOWN | VT_BYREF` or `VT_DISPATCH` are accepted?